HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -733,6 +733,10 @@ def self.process_ducts(model, ducts, building, air_loop)
       if duct.space.nil? # Outside
         duct.zone = nil
         duct.zone_handle = HPXML::LocationOutside
+      elsif duct.space.is_a? OpenStudio::Model::ScheduleConstant
+        # Pass schedule and its name to duct.zone
+        duct.zone = duct.space
```

Comment:
By making changes suggested above, we should now be able to clean this up so that it's not a space being assigned to a zone (?!), where the space actually represents a schedule (?!).